fe/fe-core/src/main/java/com/starrocks/catalog/Type.java

```diff
@@ -1576,6 +1576,8 @@ public int getMysqlResultSetFieldCharsetIndex() {
             case VARCHAR:
             case HLL:
             case BITMAP:
+            // Because mysql does not have a large int type, mysql will treat it as hex after exceeding bigint
+            case LARGEINT:
```

Comment:
Could you please support `JSON` by the way?

Reply:
Sure